src/framework/global/containers.h

```diff
@@ -61,6 +63,7 @@ inline T value(const std::vector<T>& vec, size_t idx)
         return nullptr;
     }
 
+    UNREACHABLE;
     return T();
```

Comment:
it can be reachable, I think better add 'else`

Reply:
Hmm, MSVC apparently thinks it is not? And an else branch would equally (be reported as) unreachable?

No, apparently not, so I'll go for the `else`